internal/reconcilers/certified_image_stream.go

```diff
@@ -44,6 +45,13 @@ func (r *CertifiedImageStreamReconciler) Reconcile(ctx context.Context, pipeline
 	stream := newImageStream(key)
 	if objects.IsObjectFound(ctx, r.Client, key, stream) {
 		log.Info("existing certified image stream found")
+
+		// setting owner reference on ImageStream CR, so CR gets garbage collected on OperatorPipeline deletion.
```

Comment:
Is it worth checking for an owner reference with `HasControllerReference` before attempting to set / update owner references ?

Reply:
I don't think this really buys us anything, since there isn't another controller that could own this. We don't do this check in other reconcilers.